_posts/2016-09-22-maturing-the-security-project.markdown

```diff
@@ -0,0 +1,162 @@
+[questions](http://lists.openstack.org/pipermail/openstack-dev/2016-September/104170.html) being
+asked: _who are these guys?_ _are they an active team?_ _should they be moved outside of the big
+tent?_
+
+These are understandable questions, I feel that my [on-thread
+response](http://lists.openstack.org/pipermail/openstack-dev/2016-September/104176.html) addressed
+them for the most part. What I want to focus on in this blog post is the things that we need to do to
+be a better part of the community. To ensure that project teams and the TC are both aware of what we
+do and how we help improve security in OpenStack.
+
+We know from the feedback we've had from downstream OpenStack consumers that our work
+is valued, we need to better demonstrate that value within the OpenStack community. I think a good
+place to start is to look at the [Project Team Guide](http://docs.openstack.org/project-team-guide)
+examine what we are already doing and where we fall short. Of course this doesn't include the good
+things we do like providing CI tooling for security, threat analysis etc but it is the minimum boxes
+that we should be ticking off as a project team and that I should be driving as PTL.
+
+I want to be clear, I think that the Security Project is doing great things to enhance security
+in OpenStack. We need to become a better community player though, through doing so I expect
+new opportunities to innovate on security and create new ways to make OpenStack more secure.
+
+## Score Card
+I'm sure this isn't a new idea but I wasn't able to find much good evidence of
+other teams doing this when I had a quick search.
+
+I'm proposing a score card for the security project, to ensure we're doing all that
+we should be doing and identify those areas where we need to improve. I've based this
+on the [Project Team Guide](http://docs.openstack.org/project-team-guide)
+
+Requirement | Status | Notes |
+------------|--------|-------|
+Open Code | Achieved | All code in git and licensed appropriately
+Open Design | Achieved | All design is open to the public, conducted at summits etc
+Open Development | Achieved | We follow standard OpenStack best practice
+Open Community | Needs Improvement | We have a gap around the mailing lists that we need to address
+Public Meetings on IRC | Achieved | 1700UTC Thursdays #openstack-meeting-alt
+Project IRC channel | Achieved | #openstack-security
+Community Support Channels | Mostly Achieved | We are strong on Launchpad and IRC which is where 90% of our workload comes from however we need to pay more attention to the ML and ask.openstack.org
+Planet OpenStack | Achieved | This security blog posts to planet openstack
+Participate in Design Summits | Achieved | Regular, very well attended sessions
+Participate in Cross Project meeting | Needs Improvement | We did, for a time have people attending these meetings but not much was actively invested and over time I think they just fell off peoples calendars.
```

Comment:
This isn't accurate as Doug mentioned in IRC yesterday. The CPM is on an "as-needed" basis. This is a requirement I think Doug was discussing removing.